Update CONTRIBUTING.md for code authors/reviewers (#106)
aim-rsf/browseMetadata: CONTRIBUTING.md

- Update the README or other documentation if necessary.
- Clearly describe the changes you've made and their benefits.

## Contributing to the R Package - author or reviewr of a PR

> If your contribution involves changes to the R package itself (author or reviewer of a PR) here are some specific guidelines to follow:

Reviewer: ```suggestion
If your contribution involves changes to the R package itself (as an author or reviewer of a PR), here are some specific guidelines, assuming you are using RStudio as your editor:
```

> 1. Clone this GitHub repository and ensure all branches you need are up to date with remote.

Reviewer: ```suggestion
1. Clone this GitHub repository and ensure all branches you need are up to date with remote. You can clone it by clicking on `File` > `New Project...`, then select `Version Control`, and choose `Git`. Enter the repository URL (e.g., `https://github.com/aim-rsf/browseMetadata`), select the directory to clone the repository into, and click `Create Project`.
```

> 2. Open up R Studio and ensure you have a **Git** tab in your workspace.

Reviewer: ```suggestion
2. You should have a **Git** tab in your workspace.
```

- Update the README or other documentation if necessary.
- Clearly describe the changes you've made and their benefits.

## Contributing to the R Package - as an author or reviewer of a PR

If your contribution involves changes to the R package itself (as an author or reviewer of a PR), here are some specific guidelines, assuming you are using RStudio as your editor:

1. Clone this GitHub repository locally and ensure all branches you need are up to date with remote. 
> In R Studio, you can clone it by clicking on `File` > `New Project...`, then select `Version Control`, and choose `Git`. Enter the repository URL (e.g., `https://github.com/aim-rsf/browseMetadata`), select the directory to clone the repository into, and click `Create Project`.
2. You should have a **Git** tab in your workspace.

> 3. In this **Git** tab, move to the branch you want to make changes in (or review and test the changes of someone else).

Reviewer 2: You can also ensure the branch is up to date by running a git pull.
